bigbluebutton-html5/private/config/public/acl.yaml

```diff
@@ -0,0 +1,57 @@
+acl:
+  viewer:
+    subscriptions:
+      - 'users'
+      - 'chat' 
+      - 'cursor'
+      - 'deskshare'
+      - 'meetings'
+      - 'polls'
+      - 'presentations'
+      - 'shapes'
+      - 'slides'
+      - 'captions'
+      - 'breakouts'
+    methods:
+      - 'kickUser'
+      - 'listenOnlyToggle'
+      - 'userLogout'
+      - 'assignPresenter'
+      - 'setEmojiStatus'
+      - 'validateAuthToken'
+      - 'muteUser'
+      - 'unmuteUser'
+      - 'switchSlide'
+      - 'switchSlideMessage'
+      - 'publishVote'
+      - 'logClient'
+      - 'sendChat'
+      - 'switchSlideMessage'
+  moderator:
```

Comment:
We used to have presenter permissions which were slightly different - can switch slide but can't necessarily kick user if presenter viewer. How do you go around this now? It seems to me the use case is not covered